.github/workflows/CICD.yml

```diff
@@ -567,3 +567,31 @@ jobs:
         flags: ${{ steps.vars.outputs.CODECOV_FLAGS }}
         name: codecov-umbrella
         fail_ci_if_error: false
+  cross-compile:
+    name: Cross-Compile
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: true
+      matrix:
+        job:
+          - { toolchain: x86_64-unknown-linux-gnu, target: x86_64-unknown-linux-gnu }
+          - { toolchain: x86_64-unknown-linux-gnu, target: aarch64-unknown-linux-gnu }
+          - { toolchain: x86_64-unknown-linux-gnu, target: arm-unknown-linux-gnueabihf }
```

Comment:
could you please also add s390x? thanks

Reply:
what's the ubuntu packages for the s390x toolchain

Reply:
nvm